bin/cli.js

```diff
@@ -42,26 +43,20 @@ const parseFile = file => JSON.parse(fs.readFileSync(path.resolve(file)));
 
 const loadFiles = files => {
   const result = [];
-  files.forEach(file => {
-    if (fs.existsSync(file)) {
+  files
+    .map(file => (fs.existsSync(file + '.js') ? file + '.js' : file))
+    .filter(file => fs.existsSync(file))
```

Comment:
```js
.map(file => {
  if (fs.existsSync(file + '.js')) return file + '.js';
  else if (fs.existsSync(file)) return file;
  return null;
})
.filter(f => f !== null)
```
?

Though should we really check that it exists? I'd say to just fail if provided with non-existent file, wdyt?

Reply:
Seems reasonable